addon/components/collapsible-list.js

```diff
@@ -88,6 +89,7 @@ export default Component.extend({
       if (item.get('isExpanded') && !this._isHiding) {
         item.panelWrapper.style.height = null;
       }
+      this.triggerEvent('onAfterShow', item);
```

Comment:
The `onAfterShow` callback should probably be called inside the `if (item.get('isExpanded') && !this._isHiding) {...}` statement. That's because we want to make sure that the `_isHiding` flag is `false` in case of race conditions that could happen when users click too quickly - otherwise the `onAfterShow` callback could end up firing too many ties.

Reply:
Also I'm thinking if having just the `onShow` callback instead of adding a new `onAfterShow` would make sense, but having the  `onShow` callback execute after the `transitionend` event. Let me know what you think.

Reply:
100% on moving it inside the `if`. As far as keeping just one event that works for me, I just didn't want to change the existing API. If you don't think this would be disruptive for anyone else, I'd be happy to keep one event only.

Reply:
Actually yeah, you're right about not changing the API as it looks like it would be a backwards incompatible change. Let's just go with having the `onAfterShow` callback execute only for animated accordions/collapsibles in the `if`.